editors/ccap-temple-mode.el

```diff
@@ -147,7 +193,9 @@ OUT-DIR directory to place the generated code file."
                                 (ccap-temple-module-to-path
                                  (buffer-file-name)
                                  "scala"))))
-            (shell-command (concat purs-command scala-command)))
+            (let ((command (concat purs-command "& " scala-command)))
```

Comment:
Spliting the `&&` with only one `&` here and the other in `ccap-temple-compile-command` is a little too clever for my liking...

Perhaps change `ccap-temple-compile-command` to return the plain old command and then run it with an extra `&` only where it should run in the background?

Reply:
Yeah, that's a good idea. I thought about doing that but was feeling lazy